src/main/java/com/github/nutomic/controldlna/gui/PreferencesActivity.java

```diff
@@ -48,6 +48,7 @@
 
 	public static final String KEY_ENABLE_WIFI_ON_START = "enable_wifi_on_start";
 	public static final String KEY_INCOMING_PHONE_CALL_PAUSE = "incoming_phone_call_pause";
+	public static final String KEY_PLAYBACK_LOCAL_DEVICE = "playback_local_device";
```

Comment:
When preparing a merge request it is best to make each change on a separate branch. That way the person merging it can see just the changes relating to the specific fix and can merge the changes in the order they want. Because this request is based on top of your change to always use local device I can't accept it without also accepting that change.